mantidproject/mantid
# Updates to IN6 IDF and associated loader

The main purpose of this work was to update the IN6 IDF. The description of the new instrument geometry is contained in the [documents repository](https://github.com/mantidproject/documents/blob/master/Project-Management/ILL/Instrument_Definitions/IN6/IN6_Geometry.md).

The `LoadILL` algorithm was updated to cope with the monitor IDs being greater than the detector IDs.

The `LoadILL` algorithm was also renamed to `LoadILLTOF`, and `LoadILL` was replaced with a deprecated algorithm.

Tests for loading IN4 and IN6 data have been added. The IN5 data has also been moved to be consistent with the naming conventions at the ILL.

Note original the ticket mentioned loading monitors into a separate workspace, there is a separate ticket for this instead now - https://github.com/mantidproject/mantid/issues/17138.

**To test:**

**Files for testing** - [LoadILLTOF.zip](https://github.com/mantidproject/mantid/files/472267/LoadILLTOF.zip)
- The IDF for IN6 looks as shown in [this document](https://github.com/mantidproject/documents/blob/master/Project-Management/ILL/Instrument_Definitions/IN6/IN6_Geometry.md), when loading `164192.nxs` attached to the PR
- IDF looks sensible (e.g. similar spectra in upper/lower/middle detector banks).
- Loading IN4 data looks the same - compare screenshot in4_check.png check with loading file `076779.nxs`
- Loading IN5 data looks the same - compare screenshot in5_check.png check with loading file `IN5095934.nxs`
- `LoadILLTOF` is used by `Load` when selecting TOF data. 
- Using `LoadILL` gives a deprecated warning, and a suggestion to use `LoadILLTOF`.
- `LoadILL` has deprecated warning in the docs, `LoadILLTOF` exists.

Fixes #17065.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
